.azure_pipelines/ci-pipeline-makefile-nightly.yml

```diff
@@ -56,11 +56,12 @@ jobs:
         azureSubscription: "OSTC Lab"
         scriptType: bash
         scriptPath: $(SCRIPTS_ROOT)/run-azure-tests.sh
-        arguments:
-          $(Build.SourcesDirectory)/solutions/dotnet_azure_sdk
         addSpnToEnvironment: true
       env:
         AZURE_KEYVAULT_URL: $(AZURE_TEST_KEYVAULT_URL)
+        STANDARD_STORAGE_CONNECTION_STRING: $(STANDARD_STORAGE_CONNECTION_STRING)
+        TEST1: $(Build.SourcesDirectory)/solutions/dotnet_azure_sdk
```

Comment:
Can you try to pass tests as arguments? Otherwise, you will have to define more variables when add new tests.

Like this 
```
arguments:
$(Build.SourcesDirectory)/solutions/dotnet_azure_sdk $(Build.SourcesDirectory)/tests/azure-sdk-for-cpp
```

Reply:
Makes sense, it doesnt change the line count much though since we will be adding the same amount of lines as arguments. Will add though since I do think it is neater.